src/Jackett.Common/Definitions/zelkaorg.yml

```diff
@@ -93,7 +93,12 @@
       selector: .test > tbody > tr:has(a[href^="browse.php"])
     fields:
       title:
-        selector: td:nth-child(2) > a:nth-child(1)
+        selector: a:has(img[src^="http://img.zamunda.se/pic/download.gif"])
```

Comment:
I copied the link from the download selector. The strange thing with this tracker is, that it's domain is now zelka.org, but before it was zamunda.se and they have not changed the links. I don't think this will be a problem, but if you insist, I probably need to change download and magnet selectors too.

Reply:
yes please, might as well take this opportunity to bring the indexer up to date with current standards.
Zelka supports two domains http://zelka.org/ and  http://zamunda.se/
But it could be that the zelka.org web site has in its html hard-coded links to the zamunda.se domain.
I don't know for sure, I'm blocked from accessing the sites as I don't use VPN or proxy to get a local IP address to pass their filters.
None the less, we should make the indexer a little more bullet proof by changing any selector to avoid hard coding the site link.
Also, while you are updating, can you check if any of the categories need updating?
For example, they may have added UHD categories (such as Movies/UHD).
Thank you.

Reply:
OK, I will update it. Just found a problem with categories and freeleech.